3drepo/3drepo.io
# ISSUE #2914 - Upload containers: Upload item list items

This fixes #2914

#### Description
- Uploads are displayed in a list
- Clicking a list item shows it as selected
- You can click the "filename" header to sort by filename (Note: You'll be able to sort by other headings when I make the changes to #2698 to make the field objects flat)
- Errors in the revision tag show a red error icon with the error message in the tooltip (container name too, but this is just temporary and might change with #2943)
- Filename has a tooltip to help when long filenames are cut off with small screens


#### Test cases
- Upload items to show the list. (either using the browse button or drag and drop)
- Clicking on 'filename' heading changes list order
- Clicking an item changes its appearance
- Clicking the edit button opens the settings sidebar
- Clicking delete removes the item
- Deleting all the items reverts the modal to its original state (no list headings)
- Create errors in revision box (fewer than 2 letters, more than 120), and hover over error icon to see tooltip
